static/js/tools/choropleth/choropleth_template.tsx

```diff
@@ -15,25 +15,36 @@
  */
 
 import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import { ChoroplethMap, generateBreadCrumbs } from "./choropleth"
+
+/**
+ * Generates choropleth map from API on pageload.
+ */
+window.onload = () => {
+  ReactDOM.render(
+    React.createElement(MainPane),
+    document.getElementById("main-pane")
+  );
+
+  // Generate breadcrumbs.
+  generateBreadCrumbs();
+}
 
 class MainPane extends Component {
-  render(): JSX.Element {
+  constructor(props) {
```

Comment:
you can leave this out if nothing extra is in the constructor

Reply:
Removed